### PR TITLE
Removed uillinois.edu references

### LINKED ISSUE #7: Remove reference to uillinois email
Trainees will no longer have a uillinois email. Please update this lesson accordingly @DPI-WE/technical-associates

### PR DESCRIPTION
**Pull request by armstrca**

### CODE REVIEW

**Review by heratyian — CHANGES_REQUESTED** [review]
I'd like to strike a balance between encouraging outlook and providing some guidance for uillinois emails (since TAs in the future will likely still have them). Can you move some of the uillinois DPI email specific advice to `<aside></aside>` tags? This way it will show up as a sidenote, and not be as confusing.

File: content.md [hunk 1]
```diff
@@ -10,17 +10,17 @@ Email is crucial for professional communication. Checking it frequently is essen
 
 #### Choose an Email Client
 
-- **Desktop Clients**: Examples include [Microsoft Outlook](https://www.microsoft.com/en-us/microsoft-365/outlook/email-and-calendar-software-microsoft-outlook), [Apple Mail](https://apps.apple.com/us/app/mail/id1108187098), and [Thunderbird](https://www.thunderbird.net/en-US/). For DPI, **we recommend using Outlook**.
+- **Desktop Clients**: Examples include [Microsoft Outlook](https://www.microsoft.com/en-us/microsoft-365/outlook/email-and-calendar-software-microsoft-outlook), [Apple Mail](https://apps.apple.com/us/app/mail/id1108187098), and [Thunderbird](https://www.thunderbird.net/en-US/).
 
 ![](assets/mac-outlook.png) ![](assets/mac-email.png)
 
-- **Web Clients**: For student emails we use [webmail.illinois.edu](https://webmail.illinois.edu), however other common examples include [outlook.office.com](outlook.office.com), [mail.google.com](mail.google.com) and [mail.yahoo.com](mail.yahoo.com). This is dependent on whichever email service you are using.
```
Comment (heratyian) [inline content.md:17]: Keep this

File: content.md [hunk 2]
```diff
@@ -10,17 +10,17 @@ Email is crucial for professional communication. Checking it frequently is essen
 
 #### Choose an Email Client
 
-- **Desktop Clients**: Examples include [Microsoft Outlook](https://www.microsoft.com/en-us/microsoft-365/outlook/email-and-calendar-software-microsoft-outlook), [Apple Mail](https://apps.apple.com/us/app/mail/id1108187098), and [Thunderbird](https://www.thunderbird.net/en-US/). For DPI, **we recommend using Outlook**.
```
Comment (heratyian) [inline content.md]: keep this

**Review by heratyian — CHANGES_REQUESTED** [review]

File: content.md (hunk 1 above)
Comment (armstrca) [reply content.md:17]: But students don't have illinois.edu emails anymore, right?

Comment (heratyian) [reply content.md:17]: ah good point.

**Review by heratyian — APPROVED** [review]

**Pull request merged**